doc/userguide/devguide/codebase/contributing/github-pr-workflow.rst

```diff
@@ -0,0 +1,49 @@
+GitHub PRs workflow
```

Comment:
`GitHub PR Workflow` or `GitHub Pull Request Workflow`.

Reply:
Considering what was mentioned in the last PR, I would go with 'GitHub Pull Requests Workflow' for the title.

Reply:
Right